.tekton/collector-component-pipeline.yaml

```diff
@@ -262,6 +258,27 @@ spec:
     - name: source
       workspace: workspace
 
+  - name: apply-tags
+    params:
+    - name: IMAGE
+      value: $(tasks.build-container.results.IMAGE_URL)
+    - name: ADDITIONAL_TAGS
+      value:
+      - $(tasks.determine-image-tag.results.image-tag)-latest
```

Comment:
When (if at all) will the `-latest` and `-slim` tags be removed? Can we link a Jira issue here?

Reply:
So far, it's a [distant plan](https://redhat-internal.slack.com/archives/CFMQ5C2TT/p1716822459076269) for which I created the epic ROX-24402.

Reply:
Thanks for updating the comment.